packages/components/table/__specs__/sortable_table.spec.tsx

```diff
@@ -73,8 +79,58 @@ describe("<SortableTable />", () => {
     expect(component.asFragment()).toMatchSnapshot();
   });
 
-  describe("sorting by mouse", () => {
-    it("fires callback with sorted items", async () => {
+  it("re-renders when records change", async () => {
+    const DummyComponent: React.FC = (): React.ReactElement => {
+      const [sortedRecords, setSortedRecords] = useState<Person[]>(records);
+
+      const reverseRows = useCallback(() => {
+        setSortedRecords((records) => [...records].reverse())
+      }, [])
+
+      return (
+        <>
+          <SortableTable
+            cellConfigs={cellConfigs}
+            name="donations"
+            recordIdKey="id"
+            records={sortedRecords}
+          />
+          <a href="#" onClick={reverseRows}>Reverse Rows</a>
+        </>
+      );
+    };
+    const user = userEvent.setup();
+    const component = render(<DummyComponent />);
+
+    expect(
+      component.queryAllByRole("button").map((element) => element.textContent)
+    ).toEqual(["1", "2", "3", "4"]);
+
+    const reverseRowsButton = component.getByText("Reverse Rows");
+    await act(() => user.click(reverseRowsButton));
+
+    expect(
+      component.queryAllByRole("button").map((element) => element.textContent)
+    ).toEqual(["4", "3", "2", "1"]);
+  });
```

Comment:
Open to suggestions here... I needed a way to change the value of the prop being passed in to prove the changes that were made to the SortableTable.